pylab/website/forms.py

```diff
@@ -54,16 +55,33 @@ def clean(self):
             self.check_existing_events(title, starts)
 
 
-class VotePointsForm(forms.ModelForm):
+class ProjectPointsForm(forms.ModelForm):
+    points = forms.IntegerField(
+        widget=forms.NumberInput(attrs={'max': 99, 'min': 0, 'class': 'vote-points-input'}),
+        validators=[MinValueValidator(0)],
+    )
+
+    def __init__(self, user, voting_poll, *args, **kwargs):
+        self.user = user
+        self.voting_poll = voting_poll
+        super(ProjectPointsForm, self).__init__(*args, **kwargs)
+
+        vote_qs = Vote.objects.filter(voter=self.user, project__id=self.initial['id'])
+        if vote_qs.exists():
+            vote = Vote.objects.get(voter=self.user, project__id=self.initial['id'])
+            self.fields['points'].initial = vote.points
+
     class Meta:
-        model = Vote
-        fields = ('points',)
-        widgets = {
-            'points': forms.NumberInput(attrs={'max': 99, 'class': 'vote-points-input'}),
-        }
+        model = Project
+        fields = tuple()
```

Comment:
Because this form is not devoted for modifing Project fields, instead we need one extra field and instance object in the form. ```fields``` or ```exlucde``` must be defined, so I chose to define empty ```fields```.